web/package.json

```diff
@@ -10,16 +10,18 @@
   "scripts": {
     "start": "parcel",
     "build": "parcel build",
-    "lint": "eslint src/",
-    "fix": "prettier --write src/ && eslint --fix src/"
+    "lint": "eslint src/ && stylelint src/css",
```

Comment:
There's actually a stylelint plugin for eslint:

https://github.com/stylelint/eslint-config-stylelint

Ditto with prettier:

https://github.com/prettier/prettier-eslint

How about consolidating all linting into eslint?

Reply:
@enocom 
Trying https://github.com/stylelint/eslint-config-stylelint I found that it only adds rules to our current JS ruleset.
This doesn't add functionality to use stylelint(via eslint).
I maybe wrong, but I tried different possibilities and the only effect I achieved was that I was able to add some rules to our eslint ruleset.

The package does say:
> stylelint org's shareable config for ESLint.

So, probably it's just a config shared by the org for JS files??

Reply:
Oh I misread. It's JS only. OK.